sentry/src/main/java/io/sentry/protocol/App.java

```diff
@@ -34,7 +35,7 @@
   @SuppressWarnings("unused")
   private Map<String, Object> unknown;
 
-  public String getAppIdentifier() {
+  public @Nullable String getAppIdentifier() {
```

Comment:
this would bring better interopatibility with Kotlin null safety, but yeah, maybe around 700 annotations to be added